src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs

```diff
@@ -33,10 +34,31 @@
             :dismiss-self #(swap! state dissoc :show-perms-overlay?)}])
         [comps/SidebarButton {:style :light :margin :top :color :button-blue
                               :text "Permissions..." :icon :gear
-                              :onClick #(swap! state assoc :show-perms-overlay? true)}]])
-
+                              :onClick #(swap! state assoc :show-perms-overlay? true)}]
+        [comps/SidebarButton
+         {:style :light :margin :top :color :exception-red
+          :text "Redact" :icon :trash-can
+          :onClick
+          (fn []
+            (let [conf (js/confirm "Are you sure ?")]
+              (when conf
```

Comment:
Convert

    (fn []
      (let [conf (js/confirm "Are you sure ?")]
        (when conf

to

    #(when (js/confirm "Are you sure?")

Also note the text.  Spaces don't go before punctuation.